groups/rds/locals.tf

```diff
@@ -13,20 +13,10 @@ locals {
   placement_subnet_pattern = local.secrets.placement_subnet_pattern
   placement_vpc_pattern    = local.secrets.placement_vpc_pattern
 
-  automation_subnet_cidrs = values(zipmap(
-    values(data.aws_subnet.automation).*.availability_zone,
-    values(data.aws_subnet.automation).*.cidr_block
-  ))
-  automation_subnet_ids = values(zipmap(
-    values(data.aws_subnet.automation).*.availability_zone,
-    values(data.aws_subnet.automation).*.id
-  ))
   automation_subnet_pattern = local.secrets.automation_subnet_pattern
   automation_vpc_pattern    = local.secrets.automation_vpc_pattern
```

Comment:
Is this required? Typically the specification of the automation subnets would be if Concourse access is required, which is unlikely to be the case here.

Reply:
Automation subnet & subnets not required. The following have been removed.

**local.tf:** automation_vpc_pattern & automation_subnet_pattern
**data.tf:** "aws_vpc""automation", "aws_subnet_ids""automation", "aws_subnet""automation"